### ISSUE TITLE
[Froala V3] paste.afterCleanup event not firing

### ISSUE THREAD

**Issue by karellodewijk** [froala/wysiwyg-editor#3451]
The paste.afterCleanup event is not firing:

example:

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>title</title>
    <link rel="stylesheet" href="style.css">
    <link href="https://cdn.jsdelivr.net/npm/froala-editor@3.0.0-beta.2/css/froala_editor.pkgd.min.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/froala-editor@3.0.0-beta.2/js/froala_editor.pkgd.min.js"></script>
  </head>
  <body>
    <div id="example"></div>
  </body>
  <script>
    var editor = new FroalaEditor('#example', {events: {
      'paste.afterCleanup': (clipboard_html) => {
        console.log("You pasted: ", clipboard_html)
      }
    }})
  </script>
</html>
```

Then paste something in the froala window

expected in console: "You pasted: ", pasted content

**Comment by iernie** [comment]
This seems to be the case for both afterCleanup and beforeCleanup.

Fixing this is crucial for me updating to Froala 3.

**Comment by casualuser** [comment]
fixed in 3.0.3

**Comment by grund3g** [comment]
I have the same issue with 3.0.6. The `paste.after` works but `paste.afterCleanup`, `paste.beforeCleanup` and `paste.wordPaste` not.
